std_msgs/Header header
int32[] bb
sensor_msgs/Image img
---
bool if_succeded
